### PR TITLE
Bobby dont likes math, only dropping tables

### PR DESCRIPTION
**Pull request by RFlintstone**

### CODE REVIEW

**Comment by JonathanBout** [comment]
Relevant XKCD: https://xkcd.com/327/

File: henk/.idea/workspace.xml
Comment (JonathanBout) [inline henk/.idea/workspace.xml:1]: 😭 .idea in here 😭

**Review by JonathanBout — CHANGES_REQUESTED** [review]

**Comment by HJS-18** [comment]
LGTM ⛴️

File: henk/.idea/workspace.xml
Comment (RFlintstone) [reply henk/.idea/workspace.xml:1]: Yes 
GoodYesGIF

**Review by JonathanBout — APPROVED** [review]
lngtm

File: henk/index.js
Comment (JonathanBout) [inline henk/index.js:1]: compilation output should not be here!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Review by JonathanBout — CHANGES_REQUESTED** [review]

File: henk/index.js
Comment (RFlintstone) [reply henk/index.js:1]: smh my head

**Review by JonathanBout — APPROVED** [review]
lbtm

**Review by JonathanBout — APPROVED** [review]
lbtm (= looks better to me)

**Comment by RFlintstone** [comment]
> lbtm

Looks biased to me

**Comment by RFlintstone** [comment]
> 

BaldiGIF

**Pull request merged**